### PR TITLE
added search and filters to document files in admin

### PR DESCRIPTION
**Pull request by daniel-gray-tangent**

### CODE REVIEW

**Review by OnaMosimege — APPROVED** [review]
Looks good

File: app/general/admin.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ class DocumentFileAdmin(SimpleHistoryAdmin):
     ordering = ["title"]
     list_display = ["title", "license", "document_type", "available"]
     search_fields = ["title", "license", "document_type"]
```
Comment (friedelwolff) [inline app/general/admin.py:49]: Please limit searching to title only.

**Review by friedelwolff — CHANGES_REQUESTED** [review]

File: app/general/admin.py (hunk 1 above)
Comment (OnaMosimege) [reply app/general/admin.py]: @friedelwolff  I will get Daniel to look into this for you, thanks

Comment (daniel-gray-tangent) [reply app/general/admin.py]: @OnaMosimege,  please would you approve the changes made, thank you

Comment (OnaMosimege) [reply app/general/admin.py]: @friedelwolff I approve, please check if you're 100% with everything?

**Pull request merged**